pkg/cmd/task/delete_test.go

```diff
@@ -61,7 +61,17 @@ func TestTaskDelete(t *testing.T) {
 		),
 		tb.TaskRun("task-run-2", "ns",
 			tb.TaskRunLabel("tekton.dev/task", "task"),
-			tb.TaskRunSpec(tb.TaskRunTaskRef("task")),
+			tb.TaskRunSpec(tb.TaskRunTaskRef("task", tb.TaskRefKind(v1alpha1.NamespacedTaskKind))),
+			tb.TaskRunStatus(
+				tb.StatusCondition(apis.Condition{
+					Status: corev1.ConditionTrue,
+					Reason: resources.ReasonSucceeded,
+				}),
+			),
+		),
+		tb.TaskRun("task-run-3", "ns",
```

Comment:
Might also be helpful here to add minor comment on why a ClusterTask is included.